src/Data/Data.php

```diff
@@ -54,15 +54,22 @@ public static function handler(string $type): Handler
 		$type = strtolower($type);
 
 		// find a handler or alias
+		$alias   = static::$aliases[$type] ?? null;
 		$handler = static::$handlers[$type] ??
-				   static::$handlers[static::$aliases[$type] ?? null] ??
+				   ($alias ? static::$handlers[$alias] : null) ??
 				   null;
```

Comment:
The `?? null` after the parens doesn't work and will still result in a warning.

```suggestion
				   ($alias ? static::$handlers[$alias] ?? null : null);
```